web/src/pages/apd/activities/state-costs/NonPersonnelCost/NonPersonnelCostForm.test.js

```diff
@@ -178,11 +178,31 @@ describe('Selection removes "Administrative operations" for in MMIS apd', () =>
     await setup({}, hitechApd);
 
     expect(screen.getAllByRole('option').length).toBe(7);
+
+    const dropdown = screen.getByRole('combobox');
+
+    await waitFor(() => {
+      dropdown.click();
+    });
+
+    expect(
+      await screen.findByText('Administrative operations')
```

Comment:
Awesome, thanks for adding that! I think it helps make it clear what the test is specifically checking for